assets/js/modules/analytics/datastore/settings.test.js

```diff
@@ -392,6 +415,7 @@ describe( 'modules/analytics settings', () => {
 				registry.dispatch( STORE_NAME ).receiveGetExistingTag( null );
 				registry.dispatch( STORE_NAME ).setSettings( validSettings );
 				registry.dispatch( STORE_NAME ).setProfileID( PROFILE_CREATE );
+				registry.dispatch( CORE_FORMS ).setValues( FORM_SETUP, { profileName: 'all web site data' } );
```

Comment:
Please also add a test for `canSubmitChanges` that it requires a valid `profileName` when creating a profile is selected.

Reply:
Updated.